e2e/app/text-labels.ts

```diff
@@ -93,7 +93,7 @@ export enum LinkText {
    SeeCodePreview = 'See Code Preview',
    StayHere = 'Stay Here',
    Submit = 'Submit',
-   Update = 'Update',
+   Update = 'APPLY & RECREATE',
```

Comment:
Will this break other tests?

Reply:
Good catch. I created new `ApplyRecreate` enum value, keep `Update` unchanged.